django-verdant/requirements.txt

```diff
@@ -2,14 +2,14 @@ Django==1.9.13
 wagtail==1.6.3
 django-auth-ldap==1.2.6
 Embedly==0.5.0
-elasticsearch==1.4.0
+elasticsearch==1.6.0
```

Comment:
Probably should update redis library as well

Reply:
Appears to be working fine on production. We have no plans to use any more redis features than we current do so going to leave this as it is.